docs/build_win.md

```diff
@@ -3,61 +3,74 @@ Building Pencil2D on Windows {#build_windows}
 
-- Download the <b>%Qt Online Installer</b> from [%Qt Downloads](https://www.qt.io/download-qt-installer-oss)
-- Open up the installer, click next.
-- You have to create a free Qt account if you don't have one. Don't worry, it won't cost you a penny.
-- In the next step, choose the Qt version that matches your C++ compiler.
-  - For example, select `MSVC 2019 64-bit` if you have Visual C++ 2019 installed.
-  - If you are using Qt 6, make sure to also select its Multimedia module in the Additional Libraries section.
-  - If you have no idea what to do, select the latest `Qt 5.15.x -> MinGW 8.x` and `Developer and Design Tools -> MinGW 8.x 64-bit`.
-- Agree to the license and start the installation. It will take a long time to download all of the files, so be patient. When the installation is complete, press `Done` and it will launch Qt Creator for you.
+### Install MSVC (Optional)
+
+This step is only needed if you want to use MSVC and have not installed it yet. If you want to use MinGW or if you have built C++ applications using Visual Studio before, you can skip this step and jump ahead to the next section (**Install Qt**).
+
+1. Go to the [Visual Studio download page](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2026) and download the **Build Tools for Visual Studio 2026** from the **Tools for Visual Studio** section near the bottom of the page.
+2. Run the installer, click continue. The installer might take a few moments to get everything ready.
+3. For building Pencil2D, you'll need to select at least `Individual components → Compilers, build tools, and runtimes → MSVC Build Tools for x64/x86 (Latest)` and `Individual components → SDKs, libraries, and frameworks → Windows 11 SDK (10.#.#####)`.
+   - For a more complete installation, you can also select `Workloads → Desktop & Mobile → C++ build tools` instead, but this is not strictly necessary to build Pencil2D.
+4. Click Install and wait for the installation to complete. This might take a moment, so be patient.
+
+### Install Qt
+
+1. Download the **Qt Online Installer** from [Qt Downloads](https://www.qt.io/download-qt-installer-oss)
+2. Run the installer, click next.
+3. Log in to your Qt Account, or create a new account if you don't have one yet. Don't worry, creating a Qt account won't cost you a penny.
+4. Continue through the next few steps until you are asked for the installation folder. Make sure to select **Custom installation** and click next.
+5. In the next step, choose the latest version of Qt according to the toolchain you plan to use:
+   - If you plan to use MSVC, select `Qt → Qt #.##.# → MSVC 2022 64-bit`.
+   - If you plan to use MinGW, select `Qt → Qt #.##.# → MinGW ##.#.# 64-bit` and a matching MinGW version from the `Qt → Build Tools` section.
+   - Unless you picked Qt version 5, additionally select `Qt Multimedia` in the `Additional Libraries` subsection of your Qt version.
```

Comment:
In the current online installer, you don't have an option to install Qt5, so just omit the "Unless you picked Qt version 5..."